src/components/WwwFrame/TheFooter.vue

```diff
@@ -36,6 +36,18 @@
 					Terms of use
 				</router-link>
 			</div>
+			<ul class="hide-for-print download-app-icons">
+				<li class="app-store-icon">
+					<a :href="appStoreUrl" target="_blank">
```

Comment:
@mlibazisi please follow up with @brandonatkiva to see if he wants tracking params on these links...

prob something like:
`v-kv-track-event="['Footer', App-Store-Link', 'iOS']"`